src/vga.js

```diff
@@ -409,7 +408,66 @@ function VGAScreen(cpu, bus, vga_memory_size)
         function(addr, value) { me.vga_memory_write(addr, value); }
     );
 
+    if(!! screen_options?.use_graphical_text)
+    {
+        this.graphical_text = new GraphicalText(bus, me);
+    }
+    this.text_to_graphical_mode_flag = false;   // flag that indicates a previous text-to-graphical mode transition to screen_fill_buffer()
+
     cpu.devices.pci.register_device(this);
+
+    setTimeout(() => { me.switch_screen_mode(this.graphical_mode); }, 0);
```

Comment:
```suggestion
    setTimeout(() => { this.switch_screen_mode(this.graphical_mode); }, 0);
```

(`me` is a leftover from before arrow functions)

Reply:
Sure!